### PR TITLE
Formatting inconsistencies and update instructions

### PR DESCRIPTION
**Pull request by Landrash**
Fixed inconsistencies in formatting, address use and wording.
Removed sudo from upgrade command.

### CODE REVIEW

File: source/getting-started/installation-raspberry-pi-all-in-one.markdown [hunk 1]
```diff
@@ -42,11 +41,12 @@ The All-In-One installer script will do the following automatically:
...
 
-Windows Users - Please note that after running the installer, you will need to modify a couple settings allowing you to "switch users" to edit your configuration files. The needed change within WinSCP can be seen here: [Imgur](http://i.imgur.com/tlOljo6.jpg)
 
-To upgrade the All-In-One:
-* Login to rPi ssh pi@IPADRESS
-* Change to hass user `sudo su -s /bin/bash hass`
-* Change to virtual enviroment `source /srv/hass/hass_venv/bin/activate`
-* Update HA `sudo pip3 install --upgrade homeassistant`
+
+*Windows Users* - Please note that after running the installer, you will need to modify a couple settings allowing you to "switch users" to edit your configuration files. The needed change within WinSCP can be seen here: [Imgur](http://i.imgur.com/tlOljo6.jpg)
```
Comment (balloob) [inline source/getting-started/installation-raspberry-pi-all-in-one.markdown]: I know it's not your change but we should never allow imgur images. Our documentation need to be self contained

Comment (Landrash) [inline source/getting-started/installation-raspberry-pi-all-in-one.markdown]: Not my change but I can happily fix it.

**Comment by balloob** [comment]
Please test your changes locally. The GitHub markdown parser is more forgiving. This PR broke the formatting on https://home-assistant.io/getting-started/installation-raspberry-pi-all-in-one/ (it's no longer parsed as a list)

screen shot 2016-05-24 at 8 55 23

**Comment by Landrash** [comment]
The hassards of using the *Edit this page on GitHub* links. I'll try to do it on local instance next time.

**Comment by balloob** [comment]
It's fine if you do it via the edit this page but just make sure you check
your edits after they have been published.

For lists, make sure they are surrounded by newlines

On Tue, May 24, 2016 at 9:48 PM, Landrash <notifications@github.com> wrote:

> The hassards of using the *Edit this page on GitHub* links. I'll try to
> do it on local instance next time.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/home-assistant/home-assistant.io/pull/510#issuecomment-221472241>
>



-- 
PaulusSchoutsen.nl
It's nice to be important but it's more important to be nice.

**Pull request merged**